admin_manual/configuration_server/harden_server.rst

```diff
@@ -21,6 +38,16 @@ It is highly recommended to move the data directory (where ownCloud stores its d
 of ``/var/www``), this can be done by modifying the ``datadirectory`` switch in the configuration file. It is possible to
 do this also after an instance has been installed by moving the folder manually.
 
```

Comment:
...by moving the folder manually, and then adjusting the `'datadirectory'` parameter in `config.php`.